client/verta/verta/_deployment/endpoint.py

```diff
@@ -126,13 +126,22 @@ def _get_json_by_path(cls, conn, workspace, path):
                 return endpoint
         return None
 
+    env_vars_err_msg = "env_vars must be (string, string) dictionary"
```

Comment:
**Also**, could we move this to _inside_ `update()`? I do not think this is useful as a class attribute if it is only used in one function.
```suggestion
    env_vars_err_msg = "`env_vars` must be dictionary of str keys and str values"
```

Reply:
moved it inside update. It was a class attribute to not initialize it multiple times

Reply:
That makes sense, but I think if it is just a `str` then it is a relatively small cost, and it is more helpful to have the variable isolated to the function where it's needed.

If it was a more complex object, then it may be better to have it as a class attr